src/main/resources/lib/enonic/react4xp/asset/noStoreGetter.ts

```diff
@@ -32,13 +32,12 @@ import {buildGetter} from '/lib/enonic/static';
 //  stored in caches and can be reused while fresh. If the response becomes
 //  stale, it must be validated with the origin server before reuse.
 
-export const noStoreGetter = buildGetter({
-	//cacheControl: 'no-store, no-cache, max-age=0, must-revalidate',
-	cacheControl: 'no-store, no-cache, max-age=0',
-	etag: false, // default is true in production and false in development
-	getCleanPath: (request :Request) => {
-        const prefix = request.contextPath;
+export const noStoreGetter = (request: Request): Response => requestHandler(request, {
+	cacheControl: () => 'no-store, no-cache, max-age=0',
```

Comment:
```suggestion
	cacheControl: () => 'private, no-store, no-cache, max-age=0',
```

`private` because CDNs don't care about anything else

Reply:
Added